### PR TITLE
Updated candidate journey

### PR DESCRIPTION
**Pull request by sofer**
Consolidated a couple of steps; brought language more into line with current apply page.

### CODE REVIEW

File: src/apprenticeship-guide/candidate-journey.md [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
-1. The candidate completes an application in the application window, which opens three times each year;
-1. The applicant is short-listed typically from over 50 applicants who have completed the course entry requirements (and over 1,200 candidates who have expressed interest in the programme in the previous year) for an interview and technical test, along with up to 31 other applicants;
-1. The candidate is offered a place on our pre-apprenticeship training programme along with 15 other successful candidates;
-1. After securing a place on the programme, the candidate joins a 12-week pre-apprenticeship training;
-1. Towards the end of the pre-apprenticeship, the candidates interview with employers;
-1. After the candidate accepts an apprenticeship offer, the apprentice joins the company and starts the 12-week  block training with us;
-1. After the block training, the apprentice joins their employer on a full-time basis;
+1. Once an eligible candidate completes an expression of interest, they are invited to join our Discord server, where we encourage them to actively participate in our online community and learn collaboratively with each other;
```
[the remark this answers is not in the document]
Comment (sofer) [reply src/apprenticeship-guide/candidate-journey.md:5]: We may be introducing additional steps. @yvonne?

File: src/apprenticeship-guide/candidate-journey.md [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
+1. They start work on our application requirements, which typically take at least three months to complete;
+1. When the candidate is ready, they complete an application to our 12-week pre-apprenticeship programme.
```
[the remark this answers is not in the document]
Comment (sofer) [reply src/apprenticeship-guide/candidate-journey.md:7]: *"Pre-apprenticeship"* is a name in common use for [this sort of programme](https://www.google.com/search?q=pre-apprenticeships). Arguably, giving it a different name would introduce more confusion than it removes. Any suggestions for a different name? Alternatively, we could for clarification add the wording, *"during which they are prepared for interviews with employers"*.

[the remark this answers is not in the document]
Comment (oliverjam) [reply src/apprenticeship-guide/candidate-journey.md:7]: If it's a common term I agree we should use it. I like the idea of the extra clarification

File: src/apprenticeship-guide/candidate-journey.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sofer) [reply src/apprenticeship-guide/candidate-journey.md:5]: Changing to "Once an eligible candidate is accepted into our applicant pool..." to keep it more general.

**Review by yvonne-liu — APPROVED** [review]

**Pull request merged**